content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -724,13 +724,13 @@ When you **delete** a resource this takes place in two phases.
   "kind": "ConfigMap",
   "apiVersion": "v1",
   "metadata": {
-    "finalizers": {"url.io/neat-finalization", "other-url.io/my-finalizer"},
+    "finalizers": ["url.io/neat-finalization", "other-url.io/my-finalizer"],
     "deletionTimestamp": nil,
   }
 }
 ```
 
-When a client first sends a **delete** to request removal of a resource, the `.metadata.deletionTimestamp` is set to the current time.
+When a client first sends a **delete** to request the removal of a resource, the `.metadata.deletionTimestamp` is set to the current time.
```

Comment:
```suggestion
When a client first sends a **delete** request for the removal of a resource, then the `.metadata.deletionTimestamp` is set to the current time.
```

Reply:
Hi @aj11anuj 

The suggestion you've made may be unhelpful; the change proposed in the PR is more idiomatic (at least as UK English).